step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesService.java

```diff
@@ -59,6 +59,7 @@ public void init() throws Exception {
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
     public TimeSeriesAPIResponse getTimeSeries(FetchBucketsRequest request) {
+        enrichRequestFilter(request);
```

Comment:
Shouldn't we enrich the filter for other services: discoverMeasurements, getRawMeasurementsStats and potentially getMeasurementsAttributes?